src/core/lib/debug/stats_data.c

```diff
@@ -588,6 +594,33 @@ void grpc_stats_inc_http2_send_flowctl_per_write(grpc_exec_ctx *exec_ctx,
                            grpc_stats_histo_find_bucket_slow(
                                (exec_ctx), value, grpc_stats_table_6, 64));
 }
+void grpc_stats_inc_executor_closures_per_wakeup(grpc_exec_ctx *exec_ctx,
+                                                 int value) {
+  value = GPR_CLAMP(value, 0, 1024);
+  if (value < 13) {
+    GRPC_STATS_INC_HISTOGRAM(
+        (exec_ctx), GRPC_STATS_HISTOGRAM_EXECUTOR_CLOSURES_PER_WAKEUP, value);
+    return;
+  }
+  union {
+    double dbl;
+    uint64_t uint;
+  } _val, _bkt;
+  _val.dbl = value;
+  if (_val.uint < 4637863191261478912ull) {
```

Comment:
This block (and union above) is repeated several times across this file. Could it be refactored under a function name that'd also document what's it doing? If `4637863191261478912ull` can be made into a name constant while at it, that'd be great too.

Reply:
This is code generated (there's a big banner at the head of the file)... hesitant to complicate the code generator.

Reply:
Updated the code generator to add a reminder comment every function.